packages/tooling/fast-tooling/src/web-components/css-layout/css-layout.template.ts

```diff
@@ -0,0 +1,779 @@
+    >
+        <rect x="7.5" width="3.5" height="12" rx="1" fill="white" />
+        <rect x="3" y="4" width="3.5" height="8" rx="1" fill="white" />
+        <rect y="11" width="14" height="1" fill="white" />
+    </svg>
+`;
+const alignItemsCenter = html`
+    <svg
+        width="16"
+        height="16"
+        viewBox="0 0 16 16"
+        fill="none"
+        xmlns="http://www.w3.org/2000/svg"
+    >
+        <rect x="8.5" y="2" width="3.5" height="12" rx="1" fill="white" />
+        <rect x="4" y="4" width="3.5" height="8" rx="1" fill="white" />
+        <rect x="1" y="8" width="14" height="1" fill="white" />
+    </svg>
+`;
+
+/**
+ * CSS flex-wrap icons
+ */
+const flexWrapWrap = html`
+    <svg
+        width="16"
+        height="16"
+        viewBox="0 0 16 16"
+        fill="none"
+        xmlns="http://www.w3.org/2000/svg"
+    >
+        <rect x="3" y="3" width="4" height="4" rx="1" fill="white" />
+        <rect x="9.5" y="3.5" width="3" height="3" rx="0.5" stroke="white" />
+        <rect x="3.5" y="9.5" width="3" height="3" rx="0.5" stroke="white" />
+    </svg>
+`;
+const flexWrapWrapReverse = html`
+    <svg
+        width="16"
+        height="16"
+        viewBox="0 0 16 16"
+        fill="none"
+        xmlns="http://www.w3.org/2000/svg"
+    >
+        <rect x="9" y="9" width="4" height="4" rx="1" fill="white" />
+        <rect x="9.5" y="3.5" width="3" height="3" rx="0.5" stroke="white" />
+        <rect x="3.5" y="9.5" width="3" height="3" rx="0.5" stroke="white" />
+    </svg>
+`;
+const flexWrapNoWrap = html`
+    <svg
+        width="16"
+        height="16"
+        viewBox="0 0 16 16"
+        fill="none"
+        xmlns="http://www.w3.org/2000/svg"
+    >
+        <rect x="1" y="6" width="4" height="4" rx="1" fill="white" />
+        <rect x="11.5" y="6.5" width="3" height="3" rx="0.5" stroke="white" />
+        <rect x="6.5" y="6.5" width="3" height="3" rx="0.5" stroke="white" />
+    </svg>
+`;
+
+/**
+ * CSS wrap icons
+ */
+const rowGap = html`
+    <svg
+        width="12"
+        height="12"
+        viewBox="0 0 12 12"
+        fill="none"
+        xmlns="http://www.w3.org/2000/svg"
+    >
+        <path
+            fill-rule="evenodd"
+            clip-rule="evenodd"
+            d="M0 0H12V1H0V0ZM2.76923 5H9.23077V7H2.76923V5ZM12 11H0V12H12V11Z"
+            fill="white"
+        />
+    </svg>
+`;
+const columnGap = html`
+    <svg
+        width="12"
+        height="12"
+        viewBox="0 0 12 12"
+        fill="none"
+        xmlns="http://www.w3.org/2000/svg"
+    >
+        <path
+            fill-rule="evenodd"
+            clip-rule="evenodd"
+            d="M1 12L1 0H5.68249e-07L0 12H1ZM12 0L12 12H11L11 0H12ZM6.99998 2.76923V9.23077H4.99998L4.99998 2.76923H6.99998Z"
+            fill="white"
+        />
+    </svg>
+`;
+
+/**
+ * The template for the css-layout component.
+ * @public
+ */
+export const CSSLayoutTemplate = html<CSSLayout>`
+    <template>
+        <div class="root">
```

Comment:
I could be missing an underlying need for this, but from what I can tell, you technically could remove this and the ‘template’ and use ‘host:’ where your root selector is in CSS

Reply:
Nope, this was some leftover bits while I was revving that now no longer serve a purpose, I'll remove.